jartes/wikiblock
# Adding align block support

Fixes #9 

### Description
Adds support for the native alignment block option.

### Changes in this PR
- Added the `align` support flag on the `src/block.json` file.
- Added some CSS to get the align option working on the front-end. 
- Removed the `mapWidth` block attribute as the map width is now controlled by the align block option.
- Removed the `width` parameter from the `iframe`. 
- Added a CSS rule on the `src/editor.scss` file to get the proper alignment on the editor.

### Test instructions
1. If you already have this plugin installed on your site, please deactivate it and remove it. 
2. Check out this branch `feature/alignment-support`. 
3. Run the asset build: npm run build.
4. Run the zip build: npm run plugin-zip.
5. Upload the plugin zip to your test site and activate the plugin.
6. Edit or create a new post, and add a "Wikiblock" block. 
7. Add any Wikiloc trial ([example](https://es.wikiloc.com/rutas-senderismo/roca-de-sant-salvador-lelefant-miranda-de-santa-magdalena-100-cims-2-feec-ibp-82-hkg-03-04-2018-23747018)).
8. Check that the block has the option to change the block alignment. 
9. Select any alignment option and check that works. 
10. Please, check other alignment options as well. 

## Screenshots

![Markup on 2022-07-02 at 13:53:46](https://user-images.githubusercontent.com/1046943/176999626-37ffaa95-9aca-4b4b-be5b-81bb75c626ea.png)
